### PR TITLE
Rename cabal file to wreq-sb

### PR DESCRIPTION
**Pull request by sestrella**

### CODE REVIEW

**Comment by sestrella** [comment]
@jsl looks good?, we need to rename the project first before deploying to hackage

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
+dist
```
Comment (jsl) [inline .gitignore]: Let's avoid adding this if we don't need it.

**Comment by jsl** [comment]
Looks good, but I'd leave out the gitignore file for now... we don't want to change it that much from the upstream repo, just enough to get it to pass for us on 7.10. :)

**Comment by sestrella** [comment]
@jsl I'm going to merge this since I just remove the `.gitignore` file on my last push

**Pull request merged**